.template/lib/main.dart

```diff
@@ -65,10 +84,55 @@ class MyHomePage extends StatelessWidget {
             Text(
               FlutterConfig.get('SECRET'),
               style: const TextStyle(color: Colors.black, fontSize: 24),
-            )
+            ),
+            const SizedBox(height: 24),
+            ElevatedButton(
+              onPressed: () => context.go('/page2'),
```

Comment:
to be reused here...

Reply:
Fixed in 5a789646d0597aa3c6719c36c9fe667e256ec780